### ISSUE TITLE
Custom player monitor for HC

### ISSUE THREAD

**Issue by DavidFrendin** [DavidFrendin/epoch-hc#11]
Without the player monitor, the HC get's disconnected since it does not acquire characterID and other stuff on connection.

With it, it spawns in regular shore mode and is prone to death and damage, just as any regular player.

The HC should probably be in god mode without hunger/thirst and located in debug plains and in the CIV slot.

**Comment by DavidFrendin** [comment]
Moving to 1.2 Stable, as this issue can be acceptable for a RC.

**Comment by TayTayTheKiller** [comment]
Yes I simply teleport It to the top of the map way out of the way as its location saves to the database.
